python/dgl/distributed/partition.py

```diff
@@ -109,28 +109,38 @@ def _save_graphs(filename, g_list, formats=None, sort_etypes=False):
     save_graphs(filename, g_list, formats=formats)
 
 
-def _get_inner_node_mask(graph, ntype_id):
-    if NTYPE in graph.ndata:
-        dtype = F.dtype(graph.ndata["inner_node"])
-        return (
-            graph.ndata["inner_node"]
-            * F.astype(graph.ndata[NTYPE] == ntype_id, dtype)
-            == 1
+def _get_inner_node_mask(graph, ntype_id, use_graphbolt=False, gpb=None):
+    ndata = graph.node_attributes if use_graphbolt else graph.ndata
```

Comment:
we could check `isinstance(graph, gb.FusedCSCSamplingGraph)`, then we don't need `use_graphbolt`